addons/product/product_view.xml

```diff
@@ -66,60 +66,45 @@
+                                        <label for="standard_price"/>
+                                        <div name="standard_price_uom">
+                                            <field name="standard_price" class="oe_inline" widget="monetary" options="{'currency_field': 'currency_id'}"/> 
+                                            <label string="/" class="oe_inline" groups="product.group_uom"/>
+                                            <field name="uom_rel_id" class="oe_inline" groups="product.group_uom"/>
+                                        </div>
+                                        <field name="uom_id" on_change="onchange_uom(uom_id,uom_po_id)" groups="product.group_uom"/>
                                         <field name="uom_po_id" groups="product.group_uom"/>
                                     </group>
                                 </group>
-                                <separator string="Suppliers"/>
-                                <field name="seller_ids"/>
-                                <separator string="Description for Suppliers"/>
-                                <field name="description_purchase" placeholder="This note will be displayed on requests for quotation..."/>
                             </page>
                             <page string="Inventory">
-                                <group name="inventory">
-                                     <group name="status" string="Status">
-                                        <field name="state"/>
-                                        <field name="product_manager"/>
+                                <group>
+                                    <group>
+                                       <field digits="(14, 3)" name="weight" string="Weight" attrs="{'invisible':[('type', 'in', ['service', 'digital'])]}"/>
                                     </group>
-                                    <group name="weight" string="Weights" attrs="{'invisible':[('type','=','service')]}">
-                                        <field digits="(14, 3)" name="volume"/>
-                                        <field name="weight"/>
-                                        <field name="weight_net"/>
+                                    <group>
+                                        <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
                                     </group>
                                 </group>
+                                <group name="inventory" attrs="{'invisible':[('type', 'in', ['service', 'digital'])]}"/>
+                                <separator string="Suppliers"/>
+                                <field name="seller_ids"/>
                                 <group name="packaging" string="Packaging" attrs="{'invisible':[('type','=','service')]}" groups="product.group_stock_packaging" colspan="4">
```

Comment:
check for service or digital product to display packaging 
attrs="{'invisible':[('type','in',['service','digital'])]}"